azure-pipelines.yml

```diff
@@ -19,10 +19,10 @@ resources:
       ref: refs/heads/main
 
 variables:
-  DotNet6Version: 6.0.100
-  XamarinAndroidVsix: https://download.visualstudio.microsoft.com/download/pr/7372b89a-b719-426c-9916-c33cbc6c7a61/45c38957fdcacfbee95be95ee40c4f5a4cc9ace69416625ad26e2da83b176941/Xamarin.Android.Sdk-11.4.0.5.vsix
+  DotNet6Version: 6.0.200
+  XamarinAndroidVsix: https://aka.ms/xamarin-android-commercial-d17-1-windows
   # NOTE: there wasn't a public release of 16.11 for macOS
-  XamarinAndroidPkg:  https://dl.internalx.com/vsts-devdiv/Xamarin.Android/public/4941337/d16-11/7776c9f1c8fac303c3aa57867825990850be0384/xamarin.android-11.4.0.5.pkg
+  XamarinAndroidPkg:  https://aka.ms/xamarin-android-commercial-d17-0-macos
```

Comment:
I think this is the broken package that contains `Java.Interop` `0.2.0.0`, so we don't want to update to it.

Reply:
So `xamarin-android-commercial-d17-0-macos` is broken.

Reverting to `xamarin.android-11.4.0.5.pkg`.

Is there anything newer?

Reply:
The latest 17.1 build is:

https://dl.internalx.com/vsts-devdiv/Xamarin.Android/public/5673868/d17-1/19b97b7614b275a84d9e38096bd1fe0e25a494d4/xamarin.android-12.2.0.4.pkg

I don't think we have an accompanying VS Mac release that this is in, though. This is the equivalent of what we are shipping in VS Windows right now.

Reply:
using
```
XamarinAndroidVsix: https://aka.ms/xamarin-android-commercial-d17-1-windows
# NOTE: there wasn't a public release of 16.11 for macOS
XamarinAndroidPkg: https://dl.internalx.com/vsts-devdiv/Xamarin.Android/public/5673868/d17-1/19b97b7614b275a84d9e38096bd1fe0e25a494d4/xamarin.android-12.2.0.4.pkg
```